### PR TITLE
Giving builds under Zuul their own API

### PR DESCRIPTION
**Pull request by jsanemet**
This pull request is a pre-step before I start fetching tests. Before, I did not need to do much about builds, so I just got the JSON with their data and worked with that. Now I want to do some more advanced tasks, like looking for the location of the test, fetching them, parsing... so for such thing I have created a class to start working from.

### CODE REVIEW

File: cibyl/sources/zuul/apis/rest.puml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+@startuml
```
Comment (cescgina) [inline cibyl/sources/zuul/apis/rest.puml:1]: What is this file? Is it for creating uml diagrams?

Comment (jsanemet) [reply cibyl/sources/zuul/apis/rest.puml:1]: It is meant for PlantUML, PyCharm has an editor for this tool and I took the chance. The file is a quick diagram of the file that I plan to implement. Something to get ideas together.

Comment (cescgina) [reply cibyl/sources/zuul/apis/rest.puml:1]: Oh cool, I didn't know that. Maybe the docs folder is a good place to save them, or it has to be next to the code?

**Review by cescgina — APPROVED** [review]
LGTM

File: cibyl/sources/zuul/apis/rest.puml (hunk 1 above)
Comment (jsanemet) [reply cibyl/sources/zuul/apis/rest.puml:1]: No, it can be placed anywhere.

**Pull request merged**